src/HomeAssistantManager.cpp

```
@@ -937,6 +937,32 @@ void HomeAssistantManager::handleCommand(const char* topic, const char* payload)
         return;
     }
 
+    // tray_assign: loading blueprint assigns a scanned UID to a specific AMS tray
+    if (strcmp(command, "tray_assign") == 0) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)